tests/prague/eip2537_bls_12_381_precompiles/test_bls12_map_fp2_to_g2.py

```diff
@@ -73,8 +76,73 @@ def test_valid(
     )
 
 
+@pytest.mark.parametrize(
+    "input_data,expected_output,vector_gas_value",
```

Comment:
```suggestion
@pytest.mark.parametrize("expected_output", [Spec.INF_G2], ids=[""])
@pytest.mark.parametrize(
    "input_data",
```
Same as previous comment, but this should actually not run any tests because we don't have such points.